examples/twilio-chatbot/bot.py

```diff
@@ -25,10 +25,11 @@
 logger.add(sys.stderr, level="DEBUG")
 
 
-async def run_bot(websocket_client):
+async def run_bot(websocket_client, stream_sid):
     async with aiohttp.ClientSession() as session:
         transport = FastAPIWebsocketTransport(
             websocket=websocket_client,
+            stream_sid=stream_sid,
```

Comment:
This should be an argument of Twilio serializers because `FastAPIWebsocketTransport` is generic.